main.ts

```diff
@@ -198,7 +207,8 @@ export default class ReadwiseMirror extends Plugin {
     for (let bookId in library['books']) {
       const book = library['books'][bookId];
 
-      const { title, num_highlights } = book;
+      const { title } = book;
+      const num_highlights = book['highlights'].length;

```

Comment:
Can we also please destructure `highlight` with title above?

Reply:
good catch, implemented in the latest commit